Actions/ReadSecrets/ReadSecrets.ps1

```diff
@@ -50,10 +75,20 @@ try {
                 }
             }
         }
+
+        if ($updateSettingsWithValues) {
+            $outSettings.appDependencyProbingPaths | 
+            ForEach-Object {
+                if ($($_.authTokenSecret)) {
+                    $_.authTokenSecret = GetGithubSecret -secretName $_.authTokenSecret 
+                }
+            } 
+
+            $outSettings.appDependencyProbingPaths
+        }
     }
     catch {
-        OutputError -message "Error reading from GitHub Secrets. Error was $($_.Exception.Message)"
-        exit
+        throw $_

```

Comment:
The reason for the try/catch is to add some context to where the error happens and to make errors just a tad more "user" freindly.